main.go

```diff
@@ -58,6 +63,14 @@ func main() {
 	prometheus.MustRegister(vulnerabilityGauge)
 	http.Handle("/metrics", promhttp.Handler())
 
+	http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, "true")
```

Comment:
I know, but this is just in case a human is using this endpoint to test the liveness of the service. I think it does not harm to return `true`, or maybe even better `healthy`. WDYT?

Reply:
I get your point. Let's change it to `healthy`. 👍